README.md

```diff
@@ -16,14 +16,14 @@ Development builds are also available from [our website](https://pcsx2.net/downl
 # System Requirements
 
 ## Minimum
-* OS: Windows Vista SP2 or newer or GNU/Linux (32-bit or 64-bit)
+* OS: Windows 7 SP1 or newer or GNU/Linux (32-bit or 64-bit)
 * CPU: Any that supports SSE2 (Pentium 4 and up, Athlon64 and up) @ 1600 STR or better
```

Comment:
Maybe we can remove the Pentium/Athlon part.

Reply:
Core 2 duo and Athlon 64 X2 would better fit for minimum (even if those single core works).